src/utils/constants.js

```diff
@@ -15,7 +15,7 @@ export const STARTING_CARD_MAP = {
     'E': 'E5'
   }
 
-export const MIN_NUMBER_PLAYERS = 3;
+export const MIN_NUMBER_PLAYERS = 2;
```

Comment:
Question:
When creating a game, now 2 players are pre-selected. Is it possible to still have 3 players as the default?